servcheck_process.php

```diff
@@ -350,6 +350,8 @@ function register_shutdown($test_id) {
 function plugin_servcheck_get_users($results, $test, $type, $last_log) {
 	global $httperrors, $cert_expiry_days;
 
+	$servcheck_send_email_separately = read_config_option('thold_send_email_separately');
```

Comment:
Great effort Petr!
As servcheck_send_email_separately is already setup, could you please replace thold_send_email_separately with servcheck_send_email_separately.

Regards,
MSS

Reply:
One PR and 2 errors. I probably shouldn't work at night. 
Thank you for checking